website/blog/2024-05-22-latest-dbt-stability-improvement-innovation.md

```diff
@@ -0,0 +1,120 @@
+
+This breadth of testing provides early detection of any regressions that might have been introduced by our changes to dbt-core, changes by adapter maintainers, or any of their dependencies and drivers — using the exact installed versions that would be deployed to dbt Cloud. Crucially, this helps safeguard us from breaking changes in third-party software. 
+
+### Step 3: **dbt Cloud service tests**
+
+Before the new image version goes live, we ensure that all dbt changes are cross-compatible with every dbt Cloud service that depends on Core functionality, including areas such as the Cloud IDE, the Cloud CLI, scheduled job runs, CI, and connection testing. 
+
+For each dbt Cloud service, we run a testing suite that consists of:
+
+- Unit and integration specific tests to behavior of each dbt Cloud service
+- End-to-end headless browser testing for our UI-heavier applications
+- Compatibility for each adapter with that service
+
+This step provides further depth in testing the interplay between dbt Core and dbt Cloud application-specific functionality, covering cases such as linting SQL that has an ephemeral reference, or resolving [cross-project refs](/docs/collaborate/govern/project-dependencies#how-to-write-cross-project-ref) across multi-project "dbt Mesh" deployments.
+
+### Step 4: **Canary deployment**
+
+Once *all* the aforementioned tests have passed, we roll out the latest deployment to a small subset (5%) of accounts, including our own Internal Analytics project.
+
+These "canary" deployments are continually monitored against a set of precise observability metrics. Metrics we monitor include overall job error and cancellation rates to ensure they don't deviate from our expectations relative to a stable baseline. Any anomaly immediately alerts us, and we can shut off the canary in a matter of seconds, before any additional accounts are affected.
+
+### Step 5: Phased **rollout**
+
+Once the canary deployment has been proven to run stably for at least 24 hours, we mark it as eligible for all accounts to upgrade in their next scheduled deployment of dbt Cloud.
+
+<aside>
+ℹ️ Even with a robust testing, deployment, and monitoring system in place, it will never be *impossible* for a breaking change to make it through — just as in any other SaaS application.
+
+If this does happen, we commit to identifying and rolling back any breaking changes as quickly as possible. Under the new testing and deployment model in dbt Cloud, we have consistently demonstrated to have been able roll back changes in less than an hour and ship roll forward bug fixes in less than 4 hours.
+
+All incidents are retrospected to make sure we not only identify and fix the root cause(s), but also promptly put in place testing, automation, and quality gates to ensure that the same problem never happens again.
+
+</aside>
```

Comment:
@mirnawong1 Do you have an idea of how we could make this look nicer? Maybe an `:::info` callout?